### PR TITLE
Refactoring iii

### PR DESCRIPTION
**Pull request by FriederikeM**
* Create new checkbox component
* Reduce morningChecked and eveningChecked down to one function
* Fix css for weekly card

### CODE REVIEW

File: src/components/FormModal/Checkbox.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import "./Checkbox.css";
-import { FiMoon, FiSun } from "react-icons/fi";
+
```
Comment (hugh-burgess) [inline src/components/FormModal/Checkbox.js:2]: fantastic work

File: src/components/FormModal/Checkbox.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 import PropTypes from "prop-types";
+import TimeOfDayCheckbox from "./TimeOfDayCheckbox";
 
 Checkbox.propTypes = {
   name: PropTypes.string.isRequired,
   isChecked: PropTypes.bool.isRequired,
   onDayClicked: PropTypes.func.isRequired,
-  onMorningClicked: PropTypes.func.isRequired,
-  onEveningClicked: PropTypes.func.isRequired,
+  onTimeOfDayClicked: PropTypes.func.isRequired,
```
Comment (hugh-burgess) [inline src/components/FormModal/Checkbox.js:11]: vague?

File: src/components/FormModal/FormModal.js [hunk 3]
```diff
@@ -149,13 +149,14 @@ export default function FormModal({
...
+   * @param {string} timeOfDay
    */
 
-  function handleMorningClicked(name) {
+  function handleTimeOfDayClicked(name, timeOfDay) {
     const conflictingId = findConflictingProductId(
       name,
       routineData,
-      "morning",
+      timeOfDay,
```
Comment (hugh-burgess) [inline src/components/FormModal/FormModal.js:165]: i mean, morning is more specific but sure

File: src/components/Home.css [hunk 4]
```diff
@@ -65,7 +65,7 @@
 .bottle-wrapper-one {
   animation: slide-in-left 900ms cubic-bezier(0.8, 0.02, 0.54, 1.53) backwards;
   position: relative;
-  top: -20px;
+  top: -50px;
```
Comment (hugh-burgess) [inline src/components/Home.css:68]: incredible

File: src/components/WeeklyRoutinePage/WeekDayCard.css [hunk 5]
```diff
@@ -30,8 +30,8 @@
   position: relative;
   box-shadow: 0 15px 30px 5px rgba(0, 0, 0, 0.3);
   display: grid;
-  grid-template-columns: [left] minmax(178.125px, 360px) [right] minmax(
-      178.125px,
+  grid-template-columns: [left] minmax(166.25px, 360px) [right] minmax(
+      166.25px,
```
Comment (hugh-burgess) [inline src/components/WeeklyRoutinePage/WeekDayCard.css:34]: wow super specific changes

**Review by hugh-burgess — COMMENTED** [review]
amayzing work, incredibell

File: src/components/FormModal/Checkbox.js (hunk 2 above)
Comment (FriederikeM) [reply src/components/FormModal/Checkbox.js:11]: vague on purpose :D can be either morning or evening

File: src/components/FormModal/FormModal.js (hunk 3 above)
Comment (FriederikeM) [reply src/components/FormModal/FormModal.js:165]: yeah, but now it can be also evening, ya know?

File: src/components/FormModal/Checkbox.js [hunk 6]
```diff
@@ -44,30 +43,18 @@ export default function Checkbox({
...
+            weekdayName={name}
+            timeOfDay="morning"
+            checked={morning}
+          />
+          <TimeOfDayCheckbox
+            onChange={onTimeOfDayClicked}
+            weekdayName={name}
+            timeOfDay="evening"
+            checked={evening}
+          />
```
Comment (marwinburesch) [inline src/components/FormModal/Checkbox.js:57]: cool

**Review by marwinburesch — APPROVED** [review]
Excellent work Rikki! 🐜

**Pull request merged**